purchases/src/main/kotlin/com/revenuecat/purchases/teststore/TestStoreProductConverter.kt

```diff
@@ -0,0 +1,91 @@
+@file:JvmSynthetic
+
+package com.revenuecat.purchases.teststore
+
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.PurchasesErrorCode
+import com.revenuecat.purchases.PurchasesException
+import com.revenuecat.purchases.common.networking.RCBillingProductResponse
+import com.revenuecat.purchases.models.Period
+import com.revenuecat.purchases.models.Price
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.models.TestStoreProduct
+
+internal object TestStoreProductConverter {
+
+    @JvmSynthetic
+    @Suppress("LongMethod")
+    @Throws(PurchasesException::class)
+    fun convertToStoreProduct(productResponse: RCBillingProductResponse): StoreProduct {
+        val defaultPurchaseOptionId = productResponse.defaultPurchaseOptionId
+        val purchaseOptions = productResponse.purchaseOptions
+
+        val purchaseOptionKey = defaultPurchaseOptionId ?: purchaseOptions.keys.first()
+        val purchaseOption = purchaseOptions[purchaseOptionKey]!!
```

Comment:
Theoretically, it could happen that `purchaseOptionKey` is `null`, couldn't it? That would mean a crash here if I'm not mistaken

Reply:
Hmm well, it should always have at least a purchase option I believe, so it should be impossible... However, it's true it's probably better to handle this gracefully just in case 👍

Reply:
Addressed in https://github.com/RevenueCat/purchases-android/pull/2554/commits/b23b1ec0fe8a0cc25ecb99213214dce81c8250d7